src/quick-lint-js/fe/parse-expression.cpp

```diff
@@ -1511,9 +1511,22 @@ Expression* Parser::parse_expression_remainder(Parse_Visitor_Base& v,
   case Token_Type::less:
   case Token_Type::less_less:
     if (this->options_.typescript) {
+      // class C< <T>() => RetType> // Valid.
+      // class C<<T>() => RetType>  // Invalid.
+      if (prec.in_class_extends_clause &&
```

Comment:
Oh cool, I didn't know we had `in_class_extends_clause`. That makes this so easy!